gast_to_code.py

```diff
@@ -61,11 +71,27 @@ def js_varAssign(gast):
 python: py
 """
 def gast_router(gast, out_lang):
-    if gast["type"] == "root":
-        return body(gast["body"], out_lang)
+    if type(gast) == list:
+        return list_helper(gast, out_lang)
+
+    # Primitives
+    elif gast["type"] == "num":
+        return str(gast["value"])
+    elif gast["type"] == "arr":
+        return "[" + gast_router(gast["elts"], out_lang) + "]"
+    elif gast["type"] == "str":
```

Comment:
For another time, but it may cause issues if multiline strings are not differentiated.

Reply:
@g-harel can you elaborate? Not quite sure what you mean